images/linux/scripts/installers/bazel.sh

```diff
@@ -4,13 +4,18 @@
 ##  Desc:  Installs Bazel and Bazelisk (A user-friendly launcher for Bazel)
 ################################################################################
 
+source $HELPER_SCRIPTS/install.sh
+
 # Install bazel
-curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
-echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
-apt-get update -y
-apt-get install -y bazel
+bazelVersion=$(curl -s -L "https://api.github.com/repos/bazelbuild/bazel/releases" | jq -r '.[] | select(.prerelease==false).name' | sort --unique --version-sort | grep -ve "-.*" | tail -1)
+download_with_retries "https://github.com/bazelbuild/bazel/releases/download/${bazelVersion}/bazel-${bazelVersion}-installer-linux-x86_64.sh" "/tmp" "bazel-installer.sh"
+chmod +x /tmp/bazel-installer.sh
+sudo /tmp/bazel-installer.sh
+
+bazeliskVersion=$(curl -s -L "https://api.github.com/repos/bazelbuild/bazelisk/releases" | jq -r '.[] | select(.prerelease==false).name' | sort --unique --version-sort | grep -ve "-.*" | tail -1)
 
-# Install bazelisk
-npm install -g @bazel/bazelisk
+download_with_retries "https://github.com/bazelbuild/bazelisk/releases/download/${bazeliskVersion}/bazelisk-linux-amd64" "/tmp" "bazelisk"
```

Comment:
Why do we replace npm installation?

Reply:
I wanted to make bazelisk independent from the node version selected by nvm